server.js

```diff
@@ -4844,6 +4845,70 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// wordpress theme rating integration.
+// example: https://img.shields.io/wordpress/theme/r/hestia.svg for https://wordpress.org/themes/hestia
+camp.route(/^\/wordpress\/theme\/r\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var theme = match[1];  // eg, `hestia`.
+  var format = match[2];
+  var apiUrl = 'https://api.wordpress.org/themes/info/1.1/?action=theme_information&request[slug]=' + theme;
```

Comment:
Are theme names always alphanumeric? It's probably safer to use the URL-encode `theme`, or better yet, use the `query` option to `request`. Some of the other services do this.